crates/ruff/src/rules/flake8_bandit/helpers.rs

```diff
@@ -17,7 +18,6 @@ pub fn string_literal(expr: &Expr) -> Option<&str> {
     }
 }
 
-// Maybe use regex for this?
 pub fn matches_password_name(string: &str) -> bool {
     PASSWORD_CANDIDATE_REGEX.is_match(string)
```

Comment:
I suppose we could lowercase `string` here instead of adding `(?i)` in the regex; former would require keeping the regex and the code here in sync, therefore opted to do the latter.

Reply:
Let me benchmark them...